js/main.js

```diff
@@ -100,3 +120,25 @@ function checkEmptyList(event) {
     $noEntriesText.className = 'no-entries hidden';
   }
 }
+
+$list.addEventListener('click', function (event) {
+  if (event.target.matches('i')) {
+    $entriesListView.className = 'view hidden';
+    $newEntriesView.className = 'view';
+    var currentEntry = event.target.closest('li').getAttribute('entryid');
+    currentEntry = Number.parseInt(currentEntry);
+
+    for (var entriesIndex = 1; entriesIndex < data.entries.length + 1; entriesIndex++) {
+      if (currentEntry === entriesIndex) {
```

Comment:
Why are you comparing against the current index? You should be comparing against the id of the current entry.